scene/gui/base_button.cpp

```diff
@@ -379,7 +379,7 @@ BaseButton::DrawMode BaseButton::get_draw_mode() const {
 		bool pressing;
 		if (status.press_attempt) {
 
-			pressing = status.pressing_inside;
+			pressing = (status.pressing_inside) || (0 != (option_flags & OPTION_FLAG_KEEP_PRESSED_OUTSIDE));
```

Comment:
This habit usually used to prevent from aсcidental assignment, `cond = test_value`.
`"".equals(condition)` in java prevents NPE.

Reply:
Ok, I'll change that. And yes, it's meant to prevent accidental assignments that are difficult to spot in code reviews, but I understand it's not intuitive at all when you're not used to it.

Reply:
@eklmv  exactly :)